src/wam2layers/examples.py

```diff
@@ -0,0 +1,58 @@
+"""Functions for downloading example cases."""
+
+import io
+import zipfile
+from pathlib import Path
+from typing import Callable
+
+import requests
+
+
+def extract_from_4TU_archive(download_link, target_dir):
+    # See https://stackoverflow.com/a/14260592
+    response = requests.get(download_link, stream=True)  # TODO stream necessary or not?
+    response.raise_for_status()
+    z = zipfile.ZipFile(
+        io.BytesIO(response.content)
+    )  # TODO should I use context managers here?
+    z.extractall(target_dir)
+
+
+# TODO move to test suite once actual example cases work?
+def test_4tu_extract() -> None:
+    """Download random example dataset for testing."""
+    download_link = "https://data.4tu.nl/ndownloader/items/4e291b8f-a37e-4378-8ca6-954a44fdc8fb/versions/2"
+    target_directory = Path.cwd() / "example_random"
+
+    print(f"Downloading and extracing random test dataset to {target_directory}")
+    extract_from_4TU_archive(download_link, target_directory)
+
+
+def download_input_volta() -> None:
+    # TODO: replace with published link
+    download_link = "https://data.4tu.nl/ndownloader/items/bbe10a2a-39dc-4098-a69f-0f677d06ecdd/versions/draft"
+    target_directory = Path.cwd() / "example_volta"
+
+    print(
```

Comment:
The download is 9GB, so it might be good to warn the user that it can take some time (as there is no progress bar).

If the only way that this function is run is through the CLI, `click.echo` is recommended over `print`.

Reply:
Now it logs a message for every file, so you see the progress. Changed to logging, don't want the CLI/click to seep through to the library code.